README.md

```diff
@@ -82,6 +80,4 @@ Give a ⭐️ if you like this project!
 
```

Comment:
- [ ] Kindly check on your getting started section and remove the last setup as you already have another setup.